tests/state_test.go

```diff
@@ -58,21 +62,64 @@ func TestState(t *testing.T) {
 	st.skipLoad(`^stRandom2/randomStatetest642.json`)
 
 	st.walk(t, stateTestDir, func(t *testing.T, name string, test *StateTest) {
-		for _, subtest := range test.Subtests() {
-			subtest := subtest
-			key := fmt.Sprintf("%s/%d", subtest.Fork, subtest.Index)
-			name := name + "/" + key
-			t.Run(key, func(t *testing.T) {
-				if subtest.Fork == "Constantinople" {
-					t.Skip("constantinople not supported yet")
+		execStateTest(t, st, test, name, []string{"Constantinople"})
+	})
+}
+
+// TestExecutionSpecState runs the test fixtures from execution-spec-tests.
+func TestExecutionSpecState(t *testing.T) {
+	common.RelaxPrecompileRangeForTest(true)
+	enableTestExecutionSpecState(true)
```

Comment:
To not affect other tests:

```suggestion
	common.RelaxPrecompileRangeForTest(true)
	enableTestExecutionSpecState(true)
	defer common.RelaxPrecompileRangeForTest(false)
	defer enableTestExecutionSpecState(false)
```

Reply:
This doesn't seem to work because defer is run before all tests are finished.

Reply:
Then can you pass the variable via argument?
I think there needs to be two entrypoints:
```
func TestStateWithKaiaCoreTests() { execStateTest(..., simulate=false) }
func TestStateWithEthereum() { execStateTest(..., simulate=true) }
```